Dockerfile

```diff
@@ -23,8 +22,8 @@ RUN go build -a -installsuffix cgo -o secrets-provider ./cmd/secrets-provider
 FROM busybox
 
 # =================== MAIN CONTAINER ===================
-FROM scratch as secrets-provider
-MAINTAINER CyberArk Software, Ltd.
+FROM alpine:3.11 as secrets-provider
```

Comment:
why not use `latest`?

Reply:
because I don't like the idea of automatically pulling latest for the same reason you thought; risky. But open to `latest` if you think this is the way to go

Reply:
i think that our nightly automation is good enough but i can live with 3.11.